e2e/cypress/integration/scroll/channel_scroll_spec.js

```diff
@@ -33,6 +36,103 @@ describe('Scroll', () => {
         });
     });
 
+    it('MM-T2365 Scrolling in the channel is disabled when emoji picker is open(does not affect mobile apps or browser in mobile view)', () => {
+        const firstMessageInCenter = '<<This is the first post>>';
+        const lastMessageInCenter = '<<This is the last post>>';
+        const firstMessageInRhs = '<<This is the first comment>>';
+        const lastMessageInRhs = '<<This is the last comment>>';
+
+        // # Post a starting message
+        cy.postMessage(firstMessageInCenter);
+
+        // # Post as few as messages so that scroll bar appears in the center
+        Cypress._.times(1, () => {
+            cy.uiPostMessageQuickly(MESSAGES.HUGE);
+        });
+
+        // # Post the last message
+        cy.postMessage(lastMessageInCenter);
+
+        // # scoll to top in center
+        cy.get('div.post-list__dynamic').should('be.visible').scrollTo('top', {duration: TIMEOUTS.ONE_SEC});
+
+        // * Verify we scolled to top
+        cy.findByText(firstMessageInCenter).should('exist').and('be.visible');
+        cy.findByText(lastMessageInCenter).should('exist').and('not.be.visible');
+
+        // # Scroll back to bottom
+        cy.get('div.post-list__dynamic').should('be.visible').scrollTo('bottom', {duration: TIMEOUTS.ONE_SEC});
+
+        // * Verify we scolled to bottom
+        cy.findByText(firstMessageInCenter).should('exist').and('not.be.visible');
+        cy.findByText(lastMessageInCenter).should('exist').and('be.visible');
+
+        // # Open emoji picker for the last message
+        cy.getLastPostId().then((postId) => {
+            cy.clickPostReactionIcon(postId, 'CENTER');
+        });
+
+        // # Now try to scroll in the center
+        cy.get('#post-list').scrollTo('top', {ensureScrollable: false, duration: TIMEOUTS.ONE_SEC});
+
+        // * Verify that scrolling didnt happen
+        cy.findByText(firstMessageInCenter).should('exist').and('not.be.visible');
+        cy.findByText(lastMessageInCenter).should('exist').and('be.visible');
```

Comment:
Here too  in center although in app scrolling insnt happening but in cypress it is scrolling